test/y2storage/device_test.rb

```diff
@@ -87,36 +87,14 @@
     end
   end
 
-  describe "#detect_resize_info" do
-    let(:probed) { double(Y2Storage::Devicegraph) }
-    let(:probed_partition) { double(Y2Storage::Partition, storage_detect_resize_info: resize_info) }
-    let(:resize_info) { double(Y2Storage::ResizeInfo) }
-    let(:wrapped_partition) { double(Storage::Partition, exists_in_probed?: in_probed, sid: 444) }
-
-    subject(:staging_partition) { Y2Storage::Partition.new(wrapped_partition) }
-
-    before do
-      allow(Storage).to receive(:to_partition) do |object|
-        object
-      end
-      allow(Y2Storage::StorageManager.instance).to receive(:probed).and_return probed
-    end
-
-    context "if the device does not exist in probed" do
-      let(:in_probed) { false }
-
-      it "returns nil" do
-        expect(staging_partition.detect_resize_info).to be_nil
-      end
-    end
-
-    context "if the device exists in probed" do
-      let(:in_probed) { true }
-
-      it "returns the resize info from the equivalent partition in probed" do
-        expect(probed).to receive(:find_device).with(444).and_return probed_partition
-        expect(staging_partition.detect_resize_info).to eq resize_info
-      end
+  describe "#can_resize?" do
+    let(:resize_info) { Y2Storage::ResizeInfo.new(true) }
```

Comment:
This is not the correct way of creating a `Y2Storage::ResizeInfo` object. `Y2Storage::ResizeInfo` is a wrapper around `Storage::ResizeInfo`, so the initializer receives a `Storage::ResizeInfo` object.

You cannot wrap what is not there. That's why a double like this was used before:
```ruby
let(:resize_info) { double(Y2Storage::ResizeInfo, resize_ok?: resize_ok) }
```